### DISCUSSION TITLE
Using Truenas - Docker > Variable TRANSCRIBE_OR_TRANSLATE does not seem to work.

### DISCUSSION THREAD

**Discussion by killerflood123** [McCloudS/subgen#156] [Q&A] [answered]
Hi,

It seems that the option TRANSCRIBE_OR_TRANSLATE does not work.
I used the parameter as environment var. Yet SubGen tells that the file gets transcribed. 

![image](https://github.com/user-attachments/assets/c7c43576-6ea9-402d-a12e-ed40ca66237b)

![image](https://github.com/user-attachments/assets/94bfca8a-8cb7-477c-ae61-f68226a047e8)

This variable should be supported AFAIK. how is it possible it does not get applied? I have restarted the container after changing this setting.

Thanks in advance.

**Comment by McCloudS** [comment, marked as answer]
Need more logs.  You're trying to translate the Chinese audio into an english subtitle?

The "Transcribing file" is generic language used regardless of actually transcribing or translating.
![image](https://github.com/user-attachments/assets/ad6c361c-6a25-4c1a-84af-4ec72c95d921)

**Reply by killerflood123** [reply]
Yes, but now it started it looks like it is translating anyway. Sorry to bother you with this! 

![image](https://github.com/user-attachments/assets/0b3a9d62-3f3c-4ee0-9be5-790a7ae8aa1f)

**Reply by killerflood123** [reply]
Sounds understandable. 

Thanks for the clear answer!
